samples/browser/src/main/java/org/mozilla/samples/browser/BrowserActivity.kt

```diff
@@ -46,4 +47,8 @@ open class BrowserActivity : AppCompatActivity() {
             TabsTray::class.java.name -> BrowserTabsTray(context, attrs)
             else -> super.onCreateView(parent, name, context, attrs)
         }
+
+    override fun onTrimMemory(level: Int) {
+        components.tabsUseCases.notifyOnLowMemory.invoke()
```

Comment:
We can notify the sessionmanager directly and don't need to go through a use case. It's not triggered by a user.